docs/documentation/index.html

```diff
@@ -81,79 +81,118 @@
+
+						<div class="callout callout-info">
+						<strong>Tasks:</strong>
+						<ul>
+							<li>Click <b>New Project</b> to create a new project.</li>
+							<li>or select one from the <b> Recent Projects </b> <i> (ex: TicTacToe) </i> and continue with the <a href="#UIIntro">UI Intro</a>.</li>
 						</ul>
 					</div>
 
-					<h4>Start Window</h4>
-					<p>After launching Sourcetrail you will see the <a href="#StartWindow">Start Window</a>. Now you can create your own project or choose a pre-indexed one.</p>
 					<div class="row">
 						<div class="col-sm-8 col-sm-offset-2">
 							<img src="img/start_window.png" style="width:100%;">
 						</div>
 					</div>
+
+
+					<h2>Creating a New Project</h2>
+					
+					<p>When creating a new Sourcetrail project you will use the <a href="#ProjectSetupWizard"> Project Setup Wizard </a>. This wizard splits the setup process into several subsequent steps. Depending on your project's structure and the used build system, there are different types for project setup. Choosing the correct setup method is important and can make the setup process a lot easier.</p>
+
+					<!-- <p>In the first step of the project setup, you will just need to specify name and location of your Sourcetrail project.</p> -->
+
+
 					<div class="callout callout-info">
 						<strong>Tasks:</strong>
 						<ul>
-							<li>Press <var>New Project</var> to continue creating a new project.</li>
-							<li>or select the included recent project <var>TicTacToe</var> and continue with the <a href="#UIIntro">UI Intro</a>.</li>
+							<li>Give your project a <b>Name</b> and select a <b>Location</b> for your Sourcetrail project to live.</li>
+							<li>Click <b>Add Source Group</b> to add source files to the project.</li>
 						</ul>
 					</div>
 
-
-					<h2>Creating a Project</h2>
-					<p>When creating a new Sourcetrail project you will use the <a href="#ProjectSetupWizard">Project Setup Wizard</a>. This wizard splits the setup process into several subsequent steps. Depending on your project's structure and the used build system there are different types for project setup. Choosing the right setup method is important and can make the setup process a lot easier.</p>
-
-					<p>In the first step of the project setup you will just need to specify name and location of your Sourcetrail project.</p>
-
 					<div class="row">
 						<div class="col-sm-12">
 							<img src="img/project_setup_wizard_start.png" style="width:100%;">
-						</div>
+						</div> 
 					</div>
 
+					<h3>Add Source Group</h3>
+
+					<p>Sourcetrail projects consist of multiple <b>Source Groups</b>. Each Source Group uses a certain language, a set of files and all configurations to index these files. There are different types of Source Groups for each supported programming language. Scroll past the image for detailed instructions on setting this up. </p>
```

Comment:
What about this sentence: "Creating a single Source Group is sufficient for most projects."? Maybe something like this can help reassure the reader that setting up a project in Sourcetrail is not that complicated.

Reply:
Where in the paragraph or sentence would you like me to add that?

Reply:
I think that you can just add it before the last sencente